app/api/upload/controller.js

```diff
@@ -0,0 +1,112 @@
+const accessCodes = require('../../../db/models/accessCodes');
+const cases = require('../../../db/models/cases');
+const trails = require('../../../db/models/trails');
+
+/**
+ * @method checkValid
+ *
+ * Checks if a given access token is valid for use in consent and upload requests.
+ *
+ */
+exports.checkValid = async (req, res) => {
+  const { accessCode: codeId } = req.body;
```

Comment:
I'm sure you're aware of this but want to make a note of it. Based on our conversation in slack we should be expecting the value of the `accessCode` here and not the `codeId`

Reply:
Not to confuse this PR with the conversation in Slack, but here I implemented the "value" of the access token as the primary key and named it `id`.